scenes/src/styles.scss

```diff
@@ -30,4 +30,5 @@ $theme: map_merge($theme, (foreground: $foreground));
 body {
   margin: 0;
   color: red;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Should this change get submitted separately if it's going to be needed for many scenes?